app/src/main/res/values/strings.xml

```diff
@@ -472,6 +472,7 @@
     <string name="segment_type">Segment type</string>
     <string name="sb_invalid_segment">Invalid segment start or end</string>
     <string name="contribute_to_sponsorblock">Contribute to SponsorBlock</string>
+    <string name="sponsorblock_user_id">SponsorBlock Username</string>
```

Comment:
Usernames are a seperate thing in SponsorBlock, can this be changed to "SponsorBlock UserID"?